libs/dh/metering-point/feature-search/src/lib/dh-search.component.ts

```diff
@@ -73,28 +77,56 @@ import { dhMeteringPointIdValidator } from './dh-metering-point.validator';
           }
         </watt-text-field>
       </div>
+
+      @if (mpNotFound()) {
+        <watt-empty-state size="small" icon="custom-no-results" [title]="t('noResultFound')" />
+      }
     </vater-stack>
   `,
 })
 export class DhSearchComponent {
   private router = inject(Router);
+  private activatedRoute = inject(ActivatedRoute);
   private doesMeteringPointExist = lazyQuery(DoesMeteringPointExistDocument);
 
-  searchControl = new FormControl('', [dhMeteringPointIdValidator()]);
+  searchControl = new FormControl('', {
+    validators: [dhMeteringPointIdValidator()],
+    nonNullable: true,
+  });
+
+  mpNotFound = signal(false);
+
+  constructor() {
+    this.activatedRoute.queryParamMap.pipe(takeUntilDestroyed()).subscribe((params) => {
```

Comment:
Suggestion: use a input to get the param value, like we do on other details pages.

Reply:
Good idea. Didn't know query params are also supported as `input`s but they do. Done now.